modules/aws-cloudwatch.adoc

```diff
@@ -0,0 +1,11 @@
+
+[id="aws-cloudwatch_{context}"]
+
+= AWS CLoudWatch
```

Comment:
@jonquilwilliams from a quick search it looks like the product name is "Amazon CloudWatch". You could update this line and the other references to the name in this PR to reflect this.